src/library/impl/meta_reader/signature.h

```diff
@@ -1,3 +1,4 @@
+#pragma once
 
 namespace xlang::meta::reader
 {
```

Comment:
probably too much of a breaking change to move these into a reader/writer-neutral namespace like xlang::meta::signature?

Reply:
Seems like the natural thing to do. A bit weird otherwise.

Reply:
I took a crack at this, and it turned into waaaay too much churn to swallow right now. The signature helper types are used extensively across the project in headers that assume they are in `xlang::meta::reader`.

That might be a good plan for these helpers eventually, but this parser development branch is the wrong time and place for such a sweeping refactor.